rllib/agents/sac/sac_tf_model.py

```diff
@@ -141,10 +145,13 @@ def build_q_net(self, name, observations, actions):
                 if self.dropout_rate > 0:
                     q_net.append(tf.keras.layers.Dropout(self.dropout_rate))
 
-            q_net += [
-                tf.keras.layers.Dense(
-                    units=self.q_outs, activation=None, name="{}_out".format(name))
-            ]
+            if self.vib_regularizer["enable_regularizer"]:
+                q_net += [VIB_Decoder(output_size=self.q_outs, decoder_layers=self.vib_regularizer["decoder_layers"], dropout_rate=self.dropout_rate)]
+            else:
```

Comment:
I know that adding VIB and Dropout sounds weird, but not sure if it's better to allow it anyway. Or at least comment in the config that is not possible to have both and which one has precedence.

Reply:
I am not sure if I get this but both dropout and VIB can be enabled simultaneously.